src/molecule_podman/driver.py

```diff
@@ -164,7 +168,7 @@ def name(self, value):
     @property
     def login_cmd_template(self):
         return (
-            "podman exec "
+            f"{podman_exec} exec "
```

Comment:
I've added a step that looks for the full path to the executable and uses that instead. Also it should now fail, if it can't be found. I hope this was what you meant? :)